test/ocspapitest.c

```diff
@@ -71,23 +72,27 @@ static OCSP_BASICRESP *make_dummy_resp(void)
         || !TEST_true(OCSP_basic_add1_status(bs, cid,
                                              V_OCSP_CERTSTATUS_UNKNOWN,
                                              0, NULL, thisupd, nextupd)))
-        return NULL;
+        goto err;
+    bs_out = bs;
+    bs = NULL;
```

Comment:
Presumably the point of this is so that we can free `bs` if an error occurs - but I don't see that happening anywhere

Reply:
Good catch, thanks!